bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/servlets/DMAssetPostProcessorTest.java

```diff
@@ -361,7 +361,29 @@ private void assertNewModification(String component, List<Modification> modifica
             assertEquals(actualModification.getType(), expectedModification.type);
         }
     }
+//
+    /*
+     The case when DM asset drug-n-drop on empty page. This scenario creates Image component with assigned asset.
+     Post processor should write image server url property
+     */
+    @ParameterizedTest
+    @EnumSource(
+        value = ModificationType.class,
+        names = {"CREATE", "MODIFY"}
+    )
+    public void drugNdropDMimageToEmptyPage(ModificationType modificationType) throws Exception {
+        String expectingComponent = "/content/test/jcr:content/root/container/image40";
+        prepareResource(expectingComponent, null);
+        List<Modification> modifications = prepareModifications(modificationType, "/content/test/jcr:content/root/container/image40", true);
+        servlet.process(context.request(), modifications);
+        Resource resource = context.currentResource();
+        assertNotNull(resource);
+        assertEquals(EXPECTED_IMAGE_SERVER_URL, resource.getValueMap().get(Image.PN_IMAGE_SERVER_URL, String.class));
```

Comment:
@oborysyu you need 2 different resources here, one for container, another for image